apps/condo/domains/property/schema/Property.js

```diff
@@ -217,12 +233,24 @@ const Property = new GQLListSchema('Property', {
     hooks: {
         validateInput: ({ resolvedData, existingItem, context, addValidationError }) => {
             if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
-            if (!hasDbFields(['organization', 'type', 'address', 'addressMeta'], resolvedData, existingItem, context, addValidationError)) return
+            if (!hasDbFields(REQUIRED_FIELDS, resolvedData, existingItem, context, addValidationError)) return
+
             const { dv } = resolvedData
+
             if (dv === 1) {
                 // NOTE: version 1 specific translations. Don't optimize this logic
             } else {
-                return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown \`dv\``)
+                return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown 'dv'`)
+            }
+        },
+        afterChange: async ({ operation, existingItem, updatedItem }) => {
+            const isSoftDeleteOperation = operation === 'update' && !existingItem.deletedAt && Boolean(updatedItem.deletedAt)
```

Comment:
We often use `!!` instead of explicit `Boolean()` function. Let's stick with one style and fix it in the styleguide

Reply:
I think transformations should be consistent, do we use `` for strings, or `String`? :)

Reply:
And `+` for numbers or `Number` ?

Reply:
Probably we consistently use `!!` instead of `Boolean`